package.json

```diff
@@ -63,5 +64,8 @@
     "hooks": {
       "pre-commit": "npm-run-all -ln --parallel lint test"
     }
+  },
+  "dependencies": {
+    "rollup": "4.13.1"
```

Comment:
can you elaborate what this dependency is and why it needs to be a runtime dep?

Reply:
good catch, this should be dev dependency.

The jaeger-ui uses Vite5 that uses rolloup to build the UI. The vite5 uses by default rollup 4.13.0, however the 4.13
.1 contains the fix for the IBMz build.

Reply:
then you want the resolution to be `"^4.13.1"`, not the exact pin.

Reply:
done,